orko-end-to-end-testing/cypress/integration/coins/trade.spec.js

```diff
@@ -249,52 +249,57 @@ context("Trading", () => {
         highPrice: price + 1000,
         highLimitPrice: price + 1100,
         amount: 1
-      })
-      checkCancelServerSideOrder({
-        direction: "buy",
-        amount: "1",
-        stopPrice: price + 1000,
-        limitPrice: price + 1100
+      }).then(response => {
```

Comment:
I don't think this will help. Have a read up on Cypress. Commands aren't executed sequentially, they are added to a command queue and executed in a promise chain after they have all been queued up. Every one of these methods should return immediately anyway since all they're doing is queuing the request, not executing it.

Reply:
Unless I made a mistake somewhere in the way they work... In which case that will be me misusing Cypress and the fix should be inside the functions!

Reply:
Ok,
   This works if tests are independent, but because they are all executing on the same exchange/trading pair I think they are interacting which breaks the independence of the test. Another way to address this would be to have each test execute against a different trading pair, that way they are then isolated on the exchange.